queries.sql

```diff
@@ -1,3 +1,49 @@
-/*Queries that provide answers to the questions from all projects.*/
+/* Queries that provide answers to the questions from the project requirements. */
 
-SELECT * from animals WHERE name = 'Luna';
+--Find all animals whose name ends in "mon":
+
+SELECT * FROM animals WHERE NAME LIKE '%_mon';
+
+-----------------------------------------------------------
+
+-- List the name of all animals born between 2016 and 2019:
+
+SELECT name FROM animals WHERE date_of_birth >= '2016-01-01' 
+ AND  date_of_birth <  '2019-01-01';
```

Comment:
*  Kindly adjust your query here as the [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_create_tables.md#:~:text=List%20the%20name%20of%20all%20animals%20born%20between%202016%20and%202019.) says to List the names of all animals born between 2016 and 2019, but currently, your query doesn't answer this properly if an animal was born on `2019-12-31` your query won't cover them, since your query stops checking on the 1st of January 2019, I will suggest you extend the date to the last day of `2019` eg `SELECT name FROM animals WHERE date_of_birth >= '2016/01/01' AND date_of_birth <= '2019/31/12';
`

Reply:
Hi @Davidosky007 thank you for the review.
Changes done in the queries for the date of birth.